### ISSUE TITLE
AttributeError at / 'module' object has no attribute 'View'

### ISSUE THREAD

**Issue by LeonardoGentile** [bohde/django-backbone-example#3]
It doesn't work out of the box for me, I'm on Django 1.3.1. 
I get the `"AttributeError at /  'module' object has no attribute 'View'".`

It seems something related to pystache. If I comment the entire "class View" on mustache.py file everything seems to work.

**Comment by bohde** [comment]
Which version of pystache do you have installed? The pystache dependency for this project is 0.3.1 (see `requirements.txt`). The latest pystache (0.5.2) has a different API, which isn't compatible.

**Comment by LeonardoGentile** [comment]
You were half correct. That means if I use the pystache 0.3.1 it does work correctly but only with Django 1.3.1. I have tried it with Django 1.4.1 and it throws `"ImproperlyConfigured at / Error importing template source loader django.template.loaders.filesystem.load_template_source: "'module' object has no attribute 'load_template_source'""`.
I managed this problem by changing these two lines in setting.py: 

```
'django.template.loaders.filesystem.load_template_source',
'django.template.loaders.app_directories.load_template_source',
```

with:

```
'django.template.loaders.filesystem.Loader',
'django.template.loaders.app_directories.Loader', 
```

but after this I still get `[17/Sep/2012 18:29:28] "GET /api/v1/tweet/ HTTP/1.1" 500 2880` from the console together with a 304 error for all the static files.
After all I had luck using Django 1.3.1 right away, it seems to work...
